script.js

```diff
@@ -17,3 +17,335 @@ function toggleMenu() {
+          </div>
+          <div class="model-project-left">
+              <div>
+                  <p class="model-supporting-text">${projects[5].description}</p>
+              </div>
+              <div class="model-btn">
+                  <button class="model-button-p-main" type="button">See Project <img
+                          src="./Icons/Icon-see-live.png" alt=""></button>
+                  <button class="model-button-p-main" type="button">See Source <img
+                          src="./Icons/github.png" alt=""></button>
+              </div>
+          </div>
+      </div>
+  </div>`;
+    } else {
+      document.getElementById('modal').innerHTML = `<div class="modal-header">
+      <div class="title">Keeping track of hundreds of components website</div>
+      <button data-close-button class="close-button">&times;</button>
+  </div>
+  <div class="modal-body">
+      <div class="language-tags-model language-tags">
+          <ul class="tags-model">
+              <li class="model-tag tag-1">css</li>
+              <li class="model-tag tag-2">html</li>
+              <li class="model-tag-b tag-3">bootstrap</li>
+              <li class="model-tag tag-4">Ruby</li>
+          </ul>
+      </div>
+      <div class="model-main-project">
+          <div class="model-Img-Placeholder">
+          </div>
+          <div class="model-project-left">
+              <div>
+                  <p class="model-supporting-text">A daily selection of privately personalized
+                      reads; no accounts or
+                      sign-ups
+                      required. has been the industry's
+                      standard dummy text ever since the 1500s, when an unknown printer took a
+                      standard dummy
+                      text.</p>
+              </div>
+              <div class="model-btn">
+                  <button class="model-button-p-main" type="button">See Project <img
+                          src="./Icons/Icon-see-live.png" alt=""></button>
+                  <button class="model-button-p-main" type="button">See Source <img
+                          src="./Icons/github.png" alt=""></button>
+              </div>
+          </div>
+      </div>
+  </div>`;
+    }
+  });
+});
+
+// Contact form validation
+
+const submitButton = document.getElementById('send-button');
+submitButton.addEventListener('click', formSubmit);
+
+function formSubmit(event) {
+  const userName = document.getElementById('fullname').value;
+  const email = document.getElementById('email').value;
+  const message = document.getElementById('msg').value;
+  event.preventDefault();
+  if (email !== email.toLowerCase()) {
+    document.getElementById('error').classList.add('show-error');
+  } else {
+    const userData = JSON.stringify({
+      userName,
+      email,
+      message,
+    });
+    window.localStorage.setItem('userData', userData);
+    document.getElementById('contact').reset();
+  }
+}
```

Comment:
- [ ] Great job implementing the checks however there seems to be a bug and I can't seem to send the form even if I have my email in lowercase. It might be due to the scope of your `event.preventDefault()`. Kindly fix this issue 🌥️